experimental/src/main/java/org/jboss/bacon/experimental/impl/generator/BuildConfigGenerator.java

```diff
@@ -233,8 +235,26 @@ public BuildConfig copyExisting(BuildConfiguration bc, BuildConfigurationRevisio
         return buildConfig;
     }
 
+    private static void setEnvironment(BuildConfig buildConfig, Environment env, boolean useName) {
+        if (useName) {
+            if (!env.getName().equals(buildConfig.getEnvironmentName())) {
+                log.debug(
```

Comment:
is it worth to change it into info level?

Reply:
well, Kuba said that it should not bother users, but sure

Reply:
well I would say it's informing not bothering and if they don't want to be bothered they can update their environment. Thank you